src/StaticFAQ/ArticleDetail/FAQArticleNotFound.js

```diff
@@ -33,11 +34,12 @@ class ArticleNotFound extends React.Component<Props> {
     return (
       <div className="notFoundContainer">
         <div>
-          <Heading>Sorry, that article has ceased to exist</Heading>
+          <Heading>
+            <Trans t={__('smartfaq.faq.article.not_found.title')} />
+          </Heading>
           <div className="textMargin">
             <Text>
-              It may have been deleted or moved. Or it could just be a temporary
-              technical issue.
+              <Trans t={__('smartfaq.faq.article.not_found.description')} />
             </Text>
           </div>
           <Button onClick={this.goToFAQ}>Open the Help Center</Button>
```

Comment:
Why does it say `Open the Help Center` if it redirects to `faq` route? Not to the main help centre.